Cargo.toml

```diff
@@ -19,9 +19,10 @@ incremental = false
 overflow-checks = true
 
 [workspace.dependencies]
+anyhow = "1.0"
```

Comment:
I don't think we need this new dependency, or at least it should be a dev dependency.